test/cli/testutils/util.go

```diff
@@ -95,3 +96,12 @@ func FindUp(name, dir string) string {
 		curDir = newDir
 	}
 }
+
+func RandomBytes(n int) []byte {
```

Comment:
Nit: Shall we add this into `testutils/bytes.go` to avoid the everything-fits-here problem?